Sources/BraveWallet/Crypto/BuySendSwap/SwapCryptoView.swift

```diff
@@ -327,6 +365,7 @@ struct SwapCryptoView: View {
         text: $swapTokensStore.buyAmount
       )
       .keyboardType(.decimalPad)
+      .disabled(networkStore.selectedChain.coin == .sol)
```

Comment:
why do we disable this textfield when cointype is `.sol`?

Reply:
We only support entering a from amount for Solana Swap currently.